index.js

```diff
@@ -2,19 +2,35 @@ var log = require('npmlog');
 var loader = require('./lib/retrieve_package.js');
 var sym = require('./lib/symlink.js');
 var path = require('path');
+var reader = require('./lib/file_reader.js');
+var fse = require('fs-extra');
+var fs = require('fs');
+
+/* eslint-disable */
```

Comment:
Why is this linting disabled? Would be good to add a comment about the "why"

Reply:
The linter is consistently failing on the `callback` param for the `link` function, which is why I disabled it. I've added a comment.